### PR TITLE
Certificates: Documentation for the role and variables

### PR DESCRIPTION
**Pull request by ra-rau**
- Add the documentation for the role certificates and the variables.
- This is partly https://github.com/osism/issues/issues/185

Signed-off-by: Ramona Rautenberg <rautenberg@osism.tech>

### CODE REVIEW

File: roles/certificates/README.rst [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+.. zuul:rolevar:: certificates_ca_path
+   :default: /usr/local/share/ca-certificates
+
+   The path where the certificates will be stored.
+
+.. zuul:rolevar:: certificates_ca_package_name
+   :default: ca-certificates
+
+   The Debian package what is needed to install for the certificates.
```
Comment (osfrickler) [inline roles/certificates/README.rst:29]: Use "that" or "which" instead of "what".

File: roles/certificates/README.rst [hunk 2]
```diff
@@ -0,0 +1,34 @@
+An ansible role to install and update CA certificates.
+
+**Role Variables**
+
+.. zuul:rolevar:: certificates_ca
+   :default: []
+   
+   This is a list wich contains the name and the certificate.
```
Comment (osfrickler) [inline roles/certificates/README.rst:8]: s/wich/which

Also, what is the name needed for?

**Review by osfrickler — COMMENTED** [review]
Please also update doc/source/index.rst so that the role documentation will be included in the output

File: roles/certificates/README.rst (hunk 1 above)
Comment (ra-rau) [reply roles/certificates/README.rst]: Done

File: roles/certificates/README.rst (hunk 2 above)
Comment (ra-rau) [reply roles/certificates/README.rst]: Done

**Pull request merged**